activiti-cloud-service-common/activiti-cloud-services-commons-io/pom.xml

```diff
@@ -24,8 +24,9 @@
       <artifactId>spring-data-commons</artifactId>
     </dependency>
     <dependency>
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
+      <groupId>junit</groupId>
```

Comment:
I don't think this dependency is needed

Reply:
The unit test wouldn't compile without junit dependency.

Reply:
that is because the test is using the JUnit 4 while it should have been using the JUnit 5 API, it should be fixed